includes/class-sensei.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+use Sensei\Clock\Clock;
+use Sensei\Clock\Clock_Interface;
```

Comment:
Is it needed since it's only in PHPDoc?

Reply:
Static code analyzers need it :) The other option is to use the full-qualified name in the doc block.